test/upgrade/script/run.sh

```
@@ -54,6 +54,11 @@ if ! kubectl --kubeconfig=$server_kubeconfig -n clusters-service wait deploy/wor
     exit 1
 fi
 
+if [ $(kubectl --kubeconfig=$server_kubeconfig api-resources | grep -i AuthorizationPolicy | wc -l | sed 's/\ //g') -ge 1 ]; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)